imports/plugins/core/accounts/server/no-meteor/register.js

```diff
@@ -0,0 +1,134 @@
+import mutations from "./mutations";
+import queries from "./queries";
+import { registerPluginHandler } from "./registration";
+import resolvers from "./resolvers";
+import schemas from "./schemas";
+import startup from "./startup";
+
+const ENROLL_URI_BASE = "account/enroll";
+
+/**
+ * @summary Import and call this function to add this plugin to your API.
+ * @param {ReactionNodeApp} app The ReactionNodeApp instance
+ * @return {undefined}
+ */
+export default async function register(app) {
+  await app.registerPlugin({
+    label: "Accounts",
+    name: "reaction-accounts",
+    icon: "fa fa-users",
+    autoEnable: true,
+    addRolesToGroups: [{
+      allShops: true,
+      groups: ["guest", "customer"],
+      roles: ["account/verify", "reset-password", ENROLL_URI_BASE]
+    }],
+    collections: {
+      Accounts: {
+        name: "Accounts",
+        indexes: [
+          // Create indexes. We set specific names for backwards compatibility
+          // with indexes created by the aldeed:schema-index Meteor package.
+          [{ groups: 1 }, { name: "c2_groups" }],
```

Comment:
Is there no way to not use these old index names?

Reply:
If there's a way to change index names, I suppose we could do a migration. But without that, if you leave out the name and they have the old names already in the db, you get an error about trying to create the same index with a different name. There may be some way, but it would take experimentation.